addons/point_of_sale/static/src/js/screens.js

```diff
@@ -650,6 +650,7 @@ var ProductCategoriesWidget = PosBaseWidget.extend({
                     self.perform_search(self.category, searchbox.value, event.which === 13);
                 },70);
             }
+            event.stopPropagation();
```

Comment:
please add a comment to explain why you are doing this.  Just a sentence to explain that it is to prevent the main $el to use the event, or something like that.